setup.py

```diff
@@ -24,7 +24,7 @@
         "Django>=1.11.27,<2.0.0;python_version=='2.7'",
         "Django>=2.2.9,<3.0.0;python_version=='3.5'",
         "Django>=2.2;python_version>='3.6'",
-        "djangorestframework==3.9.3;python_version=='2.7'",
+        "djangorestframework==3.11.2;python_version=='2.7'",
         "djangorestframework>=3.9.3;python_version>='3.5'",
```

Comment:
djangorestframework 3.11.2 doesn't support Python 2.7.  Bump this one patch version to 3.9.4 but since this is the last version to support Python 2.7, while django-drf-filepond maintains Python 2.7 support, this will need to stay in place to support any legacy Python 2-based apps. However for the second line that applies to Python 3.5+, this should be bumped to >=3.11.2 to ensure that Python 3-based apps are using drf 3.11.2+.

```suggestion
        "djangorestframework==3.9.4;python_version=='2.7'",
        "djangorestframework>=3.11.2;python_version>='3.5'",
```